### PR TITLE
Add public traffic configuration for admin/managed servers

### PR DESCRIPTION
**Pull request by zhengchang907**
-- Add nsgNestedTemplate.json to append NSG rules for admin/managed server
-- Update mainTemplate.json to adopt NSG rules configurations
-- Update clusterTemplate.json to adopt the WebLogic load balancer port configuration
-- Add UI for the configurations
-- Remove legacy sample code

### CODE REVIEW

**Comment by zhengchang907** [comment]
@edburns Hi Ed, could you pls review this PR? It seems that I don't have the authority to add reviewers.

File: arm-oraclelinux-wls-cluster/src/main/arm/nestedtemplates/clusterTemplate.json [hunk 1]
```diff
@@ -244,6 +244,21 @@
                             "direction": "Inbound",
                             "destinationPortRanges": [ "7005" ]
                         }
+                    },
+                    {
+                        "name": "WebLogiManagedChannelPortsDenied",
```
Comment (edburns) [inline arm-oraclelinux-wls-cluster/src/main/arm/nestedtemplates/clusterTemplate.json]: Any reason why this has `WebLogi` instead of `WebLogic`?

**Review by edburns — CHANGES_REQUESTED** [review]
Fix typo, or explain why typo is necessary.

File: arm-oraclelinux-wls-cluster/src/main/arm/nestedtemplates/clusterTemplate.json (hunk 1 above)
Comment (zhengchang907) [reply arm-oraclelinux-wls-cluster/src/main/arm/nestedtemplates/clusterTemplate.json]: I missed that spot, and will fix the type, thanks

**Pull request merged**